src/cli/Cli.spec.ts

```diff
@@ -2176,6 +2183,66 @@ describe('Cli', () => {
       });
   });
 
+  it('shows help for command in browser when --help option specified', (done) => {
+    sinon.stub(Cli.getInstance().config, 'get').callsFake((settingName) => {
+      if (settingName === settingsNames.helpTarget) {
+        return 'web';
+      }
+      return null;
+    });
+
+    const browserStub = sinon.stub(browserUtil, 'open').callsFake(async (url: string) => {
+      if (url === 'https://pnp.github.io/cli-microsoft365/cmd/cli/cli-mock') {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+
+    cli
+      .execute(rootFolder, ['cli', 'mock', '--help'])
+      .then(_ => {
+        try {
+          done();
+        }
+        catch (e) {
+          done(e);
+        }
+      }, e => done(e));
+    assert(browserStub.called);
+  });
+
+  it('shows help for command in browser when -h option specified', (done) => {
```

Comment:
Let's clarify in the name of the test that we're assuming configuration to show help in the browser